### PR TITLE
Add flag binaryAsString for parquet ingestion

### PR DESCRIPTION
**Pull request by chtefi**
Hello,

This PR adds a flag "binaryAsString" to the "parquet" parser (extensions-contrib).

This is the equivalent of the Spark flag of the same name, used when you have a parquet generated by Impala. This flag tells the converter to interpret binary data as a string.

Without this patch, the strings are deserialized as an arrayish-string such as "[104, 101, 121, 32, 116..]" instead of the plain string.

### CODE REVIEW

**Comment by fjy** [comment]
@du00cs do you mind taking a look?

**Comment by du00cs** [comment]
@chtefi 👍

**Comment by du00cs** [comment]
@fjy somebody else takes a look too?

**Comment by fjy** [comment]
@chtefi if u havent done so already, can you sign the CLA @ http://druid.io/community/cla.html?

**Comment by chtefi** [comment]
@fjy I already did it a while ago (https://github.com/druid-io/druid/pull/2385)

Thanks!

File: extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/ParquetHadoopInputRowParser.java [hunk 1]
```diff
@@ -37,14 +37,17 @@
...
 {
   private final ParseSpec parseSpec;
+  private final boolean binaryAsString;
   private final List<String> dimensions;
 
   @JsonCreator
   public ParquetHadoopInputRowParser(
-      @JsonProperty("parseSpec") ParseSpec parseSpec
+      @JsonProperty("parseSpec") ParseSpec parseSpec,
+      @JsonProperty("binaryAsString") Boolean binaryAsString
```
Comment (fjy) [inline extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/ParquetHadoopInputRowParser.java]: if you use the primitive boolean here it should default to false if it is not provided

**Comment by fjy** [comment]
👍 from me

I think the +1 from the original author and the maintainer of the parquet extension (@du00cs) is the most important review here though.

**Pull request merged**